daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -835,6 +843,18 @@ func getDeviceUUID(device string) (string, error) {
 	return uuid, nil
 }
 
+func getDeviceFS(device string) (string, error) {
```

Comment:
Isn't that a duplicate of [`ProbeFsType`](https://github.com/docker/docker/blob/master/daemon/graphdriver/devmapper/mount.go#L41)?